superset/assets/package.json

```diff
@@ -80,6 +80,8 @@
     "mustache": "^2.2.1",
     "nvd3": "1.8.6",
     "parse-iso-duration": "^1.0.0",
+    "po2json": "^0.4.5",
+    "promise-polyfill": "8.0.0",
```

Comment:
I think for this to take effect it needs to be imported where you use promises. Though we do use promises already without having the polyfill I think...

Reply:
The `fetch` polyfill uses this.